test/extended/testdata/cli/pod-with-two-containers.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: doublecontainers
+  labels:
+    name: hello-openshift
+spec:
+  containers:
+  - name: hello-openshift
+    image: aosqe/hello-openshift
+    ports:
+    - containerPort: 8080
+      protocol: TCP
+    resources:
+      limits:
+        memory: 256Mi
+    terminationMessagePath: "/dev/termination-log"
+    imagePullPolicy: IfNotPresent
+    capabilities: {}
+    securityContext: {}
+  - name: hello-openshift-fedora
+    image: aosqe/fedora_base
```

Comment:
Ditto, why not just running hello-openshift twice?

Reply:
openshift or centos orgs are preferred if you look through other test data.

Reply:
Thanks very much, I've updated.